pom.xml

```diff
@@ -858,7 +858,13 @@
       <dependency>
         <groupId>org.testcontainers</groupId>
         <artifactId>testcontainers</artifactId>
-        <version>1.14.3</version>
+        <version>1.18.3</version>
```

Comment:
nit: let's define `${testcontainers.version}` to be `1.18.3` and use the var here, so we can ensure both artifact are in the same version

Reply:
done